ThunkLibs/GuestLibs/CMakeLists.txt

```diff
@@ -75,6 +79,10 @@ function(add_guest_lib NAME)
   ## Make signed overflow well defined 2's complement overflow
   target_compile_options(${NAME}-guest PRIVATE -fwrapv)
 
+  if (EXISTS "${SOURCE_LDS_FILE_ABS}")
```

Comment:
```suggestion
  # Add linker script if set
  if (EXISTS "${SOURCE_LDS_FILE_ABS}")
```

Reply:
Every thunk guest library now sets its SONAME directly with a comment in the cmake file as to why.